2020-06-01 15:40:40:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 15:41:58:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 15:42:03:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 16:15:17:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 16:24:52:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 16:29:31:在public ArrayList<String> readerFilestr(String path)中出现异常,异常信息:java.io.FileNotFoundException: saveAutoLoginState.txt (系统找不到指定的文件。)
2020-06-01 17:42:25:在public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-01 17:51:58:在public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-01 17:52:07:在public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-01 17:54:16:在public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-01 18:25:23:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-02 23:01:18:在class UserDao下的public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:06:57:在class UserDao下的public void login()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:16:46:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:16:50:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:20:16:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:20:31:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:21:54:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:21:57:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-02 23:44:07:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-02 23:45:27:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-02 23:45:30:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 00:03:07:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-03 00:03:32:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 00:05:55:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 00:06:43:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 11:32:32:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-03 11:32:32:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 11:33:52:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Operation not allowed after ResultSet closed
2020-06-03 11:40:21:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-03 11:40:43:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-03 14:34:51:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-03 14:35:05:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:java.lang.NullPointerException
2020-06-04 18:28:45:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-04 18:28:49:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 13:37:16:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-07 16:44:38:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 16:46:39:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 16:46:48:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 16:48:01:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 16:48:13:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-07 21:47:29:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-08 19:57:44:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-09 09:09:11:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-09 09:55:04:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-09 14:15:49:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-09 16:33:03:在class Userdb下的public  static boolean updateUserPwd(User user,String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'user' in 'where clause'
2020-06-09 21:11:39:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-10 16:33:08:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-16 17:11:07:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-16 23:10:24:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:54:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:54:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:55:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:55:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:56:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:56:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:57:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:57:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:58:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:58:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:59:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:11:59:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:00:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:00:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:01:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:01:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:02:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:02:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:03:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:03:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:04:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:04:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:05:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:05:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:06:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:06:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:07:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:07:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:08:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:08:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:09:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:09:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:10:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:10:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:11:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:11:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:12:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:13:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:14:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:14:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:15:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:15:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:16:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:16:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:18:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:18:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:19:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:19:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:20:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:20:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:22:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:22:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:23:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:23:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:24:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:24:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:25:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:25:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:42:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:42:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:12:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:01:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:01:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:02:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:02:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:03:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:03:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:04:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:04:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:05:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:05:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:06:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:06:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:07:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:07:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:08:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:08:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:09:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:09:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:10:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:10:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:11:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:11:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:12:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:12:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:13:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:13:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:14:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:14:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:15:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:15:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:16:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:16:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:18:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:18:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:19:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:19:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:20:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:20:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:13:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:16:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:17:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:18:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:20:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:21:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:22:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:22:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:23:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:23:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:24:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:24:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:33:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:33:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:34:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:14:34:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:33:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:33:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:34:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:34:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:35:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:35:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:36:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:36:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:37:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:37:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:38:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:38:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:39:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:39:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:40:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:40:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:41:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:41:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:42:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:42:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:49:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:50:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:51:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:52:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:53:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:54:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:54:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:55:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:55:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:56:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:56:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:57:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:57:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:58:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:58:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:15:58:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:42:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:43:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:44:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:45:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:46:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:47:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-19 12:16:48:在public class FriendsFrame下的public void run()中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-20 23:11:26:在class UserDao下的public void login(User objUser)中出现异常,异常信息:org.json.JSONException: JSONObject["result"] not found.
2020-06-20 23:31:18:在class UserDao下的public void login(User objUser)中出现异常,异常信息:org.json.JSONException: JSONObject["result"] not found.
2020-06-21 00:42:32:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 00:53:48:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:54:38:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:55:19:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:56:19:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:57:21:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 00:57:25:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 00:58:47:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:58:59:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 00:59:32:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 01:01:57:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 01:04:34:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:00:02:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:02:24:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:12:02:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:18:20:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:18:43:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:24:03:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:25:28:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:26:00:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:29:30:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:30:20:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:33:00:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:34:44:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:38:00:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:38:04:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:38:07:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:38:59:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:40:03:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:41:13:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:41:16:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:41:18:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:41:54:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:43:44:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:45:12:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:45:45:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:46:26:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:48:50:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:48:58:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:49:05:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:49:23:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 09:51:53:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:52:01:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:52:31:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 09:52:39:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 10:07:53:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 10:09:21:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:36:03:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:36:11:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:36:26:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:36:54:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:43:33:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 15:43:44:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 15:47:31:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 16:05:47:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 16:16:25:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 16:52:03:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 17:02:24:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 17:02:52:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 17:34:31:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 20:01:26:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-21 20:36:25:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 20:40:14:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 21:08:04:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 22:01:09:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2020-06-21 22:28:37:在class UserDao下的public void login(User objUser)中出现异常,异常信息:org.json.JSONException: JSONObject["result"] not found.
2020-06-22 15:01:12:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_id1' cannot be null
2020-06-22 15:04:07:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Column 'user_id1' cannot be null
2020-06-22 15:08:09:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2020-06-22 15:08:57:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2020-06-22 15:10:55:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2020-06-22 15:11:08:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2020-06-22 15:11:18:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2020-06-22 15:15:16:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: No value specified for parameter 5
2020-06-23 12:11:32:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2020-06-26 13:29:06:在class UserDao下的public void login(User objUser)中出现异常,异常信息:org.json.JSONException: JSONObject["result"] not found.
2021-02-26 19:22:42:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-26 19:22:50:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-26 19:22:50:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-26 19:22:56:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-26 19:23:19:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-26 19:23:26:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-26 19:23:32:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-26 19:56:32:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-26 20:01:35:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-26 20:02:34:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-26 20:03:14:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-02-27 18:34:13:在class Userdb下的public boolean Loginstatus(String userid)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-27 18:34:18:在class Userdb下的public static Map<String, String> findById(String id,String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-02-28 10:52:15:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: No value specified for parameter 4
2021-02-28 10:56:40:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: No value specified for parameter 4
2021-02-28 11:02:53:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: No value specified for parameter 4
2021-02-28 11:10:12:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: No value specified for parameter 4
2021-02-28 11:12:37:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'flockName' in 'field list'
2021-02-28 11:15:11:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'flockName' in 'field list'
2021-02-28 11:19:21:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'flockName' in 'field list'
2021-02-28 11:20:08:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'flockName' in 'field list'
2021-02-28 11:24:28:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'ctreaName' in 'field list'
2021-02-28 11:27:12:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.BatchUpdateException: Unknown column 'ctreaName' in 'field list'
2021-03-01 16:21:00:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:java.sql.SQLException: Illegal operation on empty result set.
2021-03-03 10:54:06:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'createName' in 'where clause'
2021-03-03 10:55:48:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'createName' in 'where clause'
2021-03-03 10:56:04:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'createName' in 'where clause'
2021-03-03 11:10:42:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:java.sql.SQLException: Column 'ctreaName' not found.
2021-03-03 11:12:30:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:java.sql.SQLException: Column 'A.ctreaName' not found.
2021-03-03 11:15:07:在class Userdb下的public static List<Map<String, String>> findAll(String sql)中出现异常,异常信息:java.sql.SQLException: Column 'A.ctreaName' not found.
2021-03-03 19:19:27:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-03-03 19:23:24:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-03-03 19:23:33:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-03-05 15:05:20:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2021-03-09 18:56:36:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.lang.NullPointerException
2021-03-09 18:57:27:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.lang.NullPointerException
2021-03-09 20:25:57:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2021-03-09 20:26:23:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2021-03-09 20:27:51:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2021-03-10 17:30:44:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-03-11 11:06:31:在class Server下的public static void main(String[] args)中出现异常,异常信息:java.net.BindException: Address already in use: Cannot bind
2021-03-12 09:28:38:在class Userdb下的public static  List<Map<String, String>> findFriends(String id,String sql)中出现异常,异常信息:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'where user_id='111'' at line 1
2021-03-14 12:24:46:在class Userdb下的public static  List<Map<String, String>> findFriends(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Illegal operation on empty result set.
2021-03-14 12:29:57:在class Userdb下的public static  List<Map<String, String>> findFriends(String id,String sql)中出现异常,异常信息:java.sql.SQLException: Illegal operation on empty result set.
2021-03-15 10:00:02:在class UserDao下的public void login(User objUser)中出现异常,异常信息:java.net.SocketTimeoutException: Receive timed out
2021-03-17 09:02:37:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:04:12:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:16:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:28:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:29:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:29:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:29:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:32:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:33:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:33:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
2021-03-17 09:48:33:在class Userdb下的public  static boolean Regisration(User user,String sql)中出现异常,异常信息:java.sql.SQLException: Column count doesn't match value count at row 1
